packages/flame/lib/src/extensions/rect.dart

```diff
@@ -37,11 +37,7 @@ extension RectExtension on Rect {
       bottomLeft.toVector2(),
     ];
   }
-}
 
-// Until [extension] will allow static methods we need to keep these functions
-// in a utility class
-class RectFactory {
```

Comment:
are we getting rid of Factories?

Reply:
No, but I put it inside the extension, like you had done for another extension.

Reply:
we should probably decide & standardize cc @wolfenrain

Reply:
There are no other separate factory classes among the extensions now, so it kind of is standardized?